packages/changed-elements-react/src/clients/ComparisonJobClient.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type {
+  ChangedElements, IComparisonJobClient, ComparisonJob, GetComparisonJobParams, GetComparisonJobResultParams,
+  PostComparisonJobParams,
+  DeleteComparisonJobParams
+} from "./IComparisonJobClient.js";
+import { callITwinApi } from "./iTwinApi.js";
+export interface ComparisonJobClientParams {
+  baseUrl: string;
+  getAccessToken: () => Promise<string>;
+}
+
+export class ComparisonJobClient implements IComparisonJobClient {
+  private static readonly _acceptHeader = "application/vnd.bentley.itwin-platform.v2+json";
+  private _baseUrl: string;
+  private _getAccessToken: () => Promise<string>;
+
+  constructor(args: ComparisonJobClientParams) {
+    this._baseUrl = args.baseUrl;
+    this._getAccessToken = args.getAccessToken;
+  }
+
+  deleteComparisonJob(args: DeleteComparisonJobParams): Promise<void> {
```

Comment:
This seems to be missing `public async`
```suggestion
  public async deleteComparisonJob(args: DeleteComparisonJobParams): Promise<void> {
```